docs/snackbar.js

```diff
@@ -0,0 +1,57 @@
+export {showSnackbar};
+
+const ANIMATION_DURATION_MS = 500;
+const SNACKBAR_DURATION_MS = 3000;
+
+// Track number of messages shown so we don't accidentally hide a later message.
+let showNumber = 0;
+
+/**
+ * Shows a snackbar notification.
+ *
+ * @param {string} msg the message to be shown
+ * @param {function} snackbarStyler applies styles to the snackbar div
+ * @param {function} iconStyler applies styles to the icon div
+ * @param {number} duration Number of milliseconds to show the message. If
+ *     non-positive, message stays up until this method is called again.
+ */
+function showSnackbar(
+    msg, snackbarStyler, iconStyler, duration = SNACKBAR_DURATION_MS) {
+  const snackbar = document.getElementById('snackbar');
+  const snackbarOriginalClasses = snackbar.className;
```

Comment:
I think that if two messages come in quickly, the second message will restore the first message's style, which is not what we want, right? Maybe just create new elements each time?

Reply:
Ah thanks for catching that!

I think adding more elements makes it more complicated so I just initialized the original style on load.